### PR TITLE
helm charts of aws-csi

### PR DESCRIPTION
**Pull request by min-zh**

### CODE REVIEW

File: src/test/aws-fsx-csi-driver/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: aws-fsx-csi-driver
+description: The Amazon FSx for Lustre Container Storage Interface (CSI) Driver implements CSI specification for container orchestrators (CO) to manage lifecycle of Amazon FSx for Lustre filesystems.
+type: application
+version: 0.1.0
+appVersion: latest
```
Comment (hlwanghl) [inline src/test/aws-fsx-csi-driver/Chart.yaml:6]: Better to specify the exact version here.

Comment (min-zh) [reply src/test/aws-fsx-csi-driver/Chart.yaml]: yes, i fix this

**Pull request merged**